DeepSpeech.ipynb

```diff
@@ -101,7 +101,11 @@
     "import importlib\n",
     "ds_importer_module = importlib.import_module('util.importers.%s' % ds_importer)\n",
     "\n",
-    "from util.website import maybe_publish\n",
+    "try:\n",
```

Comment:
The context of this PR was "make the repository friendlier to external contributors" so I bundled this in because it was in the back of my mind when I was touching the external dependencies list. I can split it out if you prefer.